cmd/admin-user-add.go

```diff
@@ -54,17 +54,19 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 EXAMPLES:
-  1. Add a new user 'foobar' to MinIO server.
+  1. Add a new user 'foobar' with id '123' and group '456' to MinIO server.
```

Comment:
Should it be "uid:123" and "gid:456" instead of just numbers?

Reply:
Changed regexp in minio and help message in this repo